sections/futures/OrderSizing/OrderSizing.tsx

```diff
@@ -113,7 +113,7 @@ const OrderSizing: React.FC<OrderSizingProps> = ({ disabled }) => {
 
 			<CustomInput
 				disabled={isDisabled}
-				right={marketAsset || 'sUSD'}
+				right={marketAsset?.replace(/^s/, '') || 'sUSD'}
```

Comment:
Maybe we could use the `getDisplayAsset` function from `utils/futures` for consistency.

Reply:
Oh duh, thanks! I was looking for something like this and completely missed it 🤦